### DISCUSSION TITLE
Parrot not integrated to conda?

### DISCUSSION THREAD

**Discussion by aldbr** [cooperative-computing-lab/cctools#3353] [Installation] [answered]
Hello,

I wanted to install the latest version of Parrot using `conda`. 
I followed the `Install From Conda` section of the [installation guide](https://cctools.readthedocs.io/en/latest/install/) and noticed that `parrot_run` was unavailable.

```bash
$ conda install -y -c conda-forge ndcctools # I am in an existing environment
...
$ parrot_run -v 
parrot_run: command not found

$ ls $HOME/miniconda3/envs/.../bin/
# I can see poncho, makeflow, but parrot_run is not present
``` 

IIUC, it has been removed from the conda package on purpose: 
https://github.com/cooperative-computing-lab/cctools/blob/master/packaging/conda/build.sh#L2

Is there any reason?
I might have read the documentation too fast, but I don't see any comment about that.

Best regards

**Comment by dthain** [comment, marked as answer]
If I recall, the issue here is that Parrot can be configured in a variety of ways that add a large number of dependencies.  And we didn't want to create a huge software burden for those using Work Queue but not Parrot.

You can still install it using the GitHub instructions:
https://cctools.readthedocs.io/en/stable/install/#install-from-github

Please give that a try. and let us know how it goes for you.

**Comment by aldbr** [comment]
Alright, thanks for the explanations.
It could be worth adding a line about this in the documentation, couldn't it?

I have been able to follow the GitHub instructions, thanks!
